generators/server/templates/src/main/java/package/repository/AuthorityRepository.java.ejs

```diff
@@ -0,0 +1,13 @@
+package <%=packageName%>.repository;
+
+import <%=packageName%>.domain.Authority;
+
+import io.micronaut.data.annotation.Repository;
+import io.micronaut.data.jpa.repository.JpaRepository;
+
+/**
+ * Micronaut Predator repository for the {@link Authority} entity.
```

Comment:
Shouldn't we stick to the "official" name Micronaut Data?

Reply:
Good catch! Fixed in latest PR